docs/canvas/canvas-elements.asciidoc

```diff
@@ -83,40 +86,25 @@ Move and resize your elements to meet your design needs.
 [[format-canvas-elements]]
 ==== Format elements
 
-Align, distribute, and reorder elements for consistency and readability across your workpad pages.
-
-Access the align, distribute, and reorder options by clicking the *Element options* icon.
-
-[role="screenshot"]
-image::images/canvas_element_options.png[]
+For consistency and readability across your workpad pages, align, distribute, and reorder elements.
 
 To align elements:
 
 . Press and hold Shift, then select the elements you want to align.
 
-. Click the , then select *Group*.
-
-. Click the *Element options* icon, then select *Alignment*.
-
-. Select the alignment option.
+. Click *Edit > Alignment*, then select the alignment option.
 
 To distribute elements:
 
 . Press and hold Shift, then select the elements you want to distribute.
```

Comment:
Can we specify here that the option is only available when at least 3 elements are selected?